c2corg_ui/templates/base.html

```diff
@@ -149,6 +151,10 @@
            vcRecaptchaServiceProvider.setSiteKey('${settings["recaptcha_site_key"]}');
          });
 
+         angular.module('app').constant('angularMomentConfig', ['moment-timezone'], {
+           timezone: 'Europe/London'
```

Comment:
If you do not set `timezone`, it will take the system timezoneß That's what we want. So why do you set it? :)